scrapli_community/cisco/aireos/cisco_aireos.py

```diff
@@ -0,0 +1,51 @@
+"""scrapli_community.cisco.aireos.cisco_aireos"""
+
+from scrapli.driver.network.base_driver import PrivilegeLevel
+from scrapli_community.cisco.aireos.async_driver import (
+    default_async_on_close,
+    default_async_on_open,
+)
+from scrapli_community.cisco.aireos.sync_driver import default_sync_on_close, default_sync_on_open
+
+DEFAULT_PRIVILEGE_LEVELS = {
+    "exec": (
+        PrivilegeLevel(
+            pattern=r"^\(Cisco Controller\) >$",
```

Comment:
Is it always `Cisco Controller` or can users set hostnames?

Reply:
Hey @carlmontanari, according to this doco (step 3):

https://www.cisco.com/c/en/us/td/docs/wireless/controller/8-5/config-guide/b_cg85/administration_of_cisco_wlc.html

_**
Note: The system prompt can be any alphanumeric string up to 31 characters. You                                                          can change it by entering the config prompt    command.
**_

I'll adjust the regex accordingly

Reply:
It's now been adjusted as requested